sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -80,7 +80,6 @@ import org.apache.spark.sql.execution.streaming.StreamingQueryWrapper
 import org.apache.spark.sql.expressions.{Aggregator, ReduceAggregator, SparkUserDefinedFunction, UserDefinedAggregator, UserDefinedFunction}
 import org.apache.spark.sql.internal.{CatalogImpl, TypedAggUtils}
 import org.apache.spark.sql.internal.ExpressionUtils.column
-import org.apache.spark.sql.protobuf.{CatalystDataToProtobuf, ProtobufDataToCatalyst}
```

Comment:
then I guess we can remove 
https://github.com/apache/spark/blob/d7f0c8fca7bda2a08325a30242f6dacf0902faec/sql/connect/server/pom.xml#L110-L115

Reply:
We still need it for tests.

Reply:
send https://github.com/apache/spark/pull/48051 to update its scope to `test`